src/makeCalculator.js

```diff
@@ -4,7 +4,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(num) {
+      this.result += num;
+    },
+    subtract(num) {
+      this.result -= num;
+    },
+    multiply(num) {
+      this.result *= num;
+    },
+    divide(num) {
+      if (num === 0) {
+        return null;
```

Comment:
Returning `null` when attempting to divide by zero might not be the best approach. Consider throwing an error or returning a specific error message to indicate that division by zero is not allowed.